pymeasure/instruments/keysight/keysightN7744C.py

```diff
@@ -0,0 +1,158 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from pymeasure.instruments import Instrument, SCPIMixin, Channel
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class MPPMChannel(Channel):
+    """A channel of the voltage source."""
+
+    wavelength = Channel.control(
+        "SENSe{ch}:POWer:WAVelength?", "SENSe{ch}:POWer:WAVelength %gNM",
+        """Control the sensor wavelength of this channel.""",
```

Comment:
In which units? I guess m or nm?

You can use the docstring format described here: https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings

e.g:
```suggestion
        """Control the sensor wavelength of this channel in nm (float strictly between 1250 an 1659).""",
```